src/awkward/operations/ak_from_feather.py

```diff
@@ -0,0 +1,76 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+__all__ = ("from_feather",)
+import awkward as ak
+from awkward._dispatch import high_level_function
+
+
+@high_level_function()
+def from_feather(
+    path,
+    *,
+    columns=None,
+    use_threads=True,
+    memory_map=False,  # storage_options?
+    generate_bitmasks=False,
+    highlevel=True,
+    behavior=None,
+):
+    """
+    Args:
+        path (str file path, or file-like object): Can be a MemoryMappedFile as
+            source, for explicitly use memory map.
+        columns (sequence): Only read a specific set of columns. If not provided,
+            all columns are read.
+        use_threads (bool): If True, parallelize reading using multiple threads.
+        memory_map (bool): If True, use memory mapping when opening file on disk,
+            when source is a string.
+        generate_bitmasks (bool): If enabled and Arrow/Parquet does not have Awkward
+            metadata, `generate_bitmasks=True` creates empty bitmasks for nullable
+            types that don't have bitmasks in the Arrow/Parquet data, so that the
+            Form (BitMaskedForm vs UnmaskedForm) is predictable.
+        highlevel (bool): If True, return an #ak.Array; otherwise, return
+            a low-level #ak.contents.Content subclass.
+        behavior (None or dict): Custom #ak.behavior for the output array, if
+            high-level.
+
+    Reads an Feather file as an Awkward Array (through pyarrow).
+
+        >>> import awkward as ak
+
+        # Result is an Awkward Array.
+        >>> array = ak.from_feather("file_name.feather")
+        <Array [[1.1, 2.2, 3.3], [], [4.4, 5.5]]
+            type='3 * var * float64'>
```

Comment:
Examples in docstrings generally don't have the "import awkward as ak" part (because otherwise, they'd all have that) and a line that assigns to a variable wouldn't return the value of that variable, as you've shown it above. To show what this does and have it be a realistic terminal log, just don't assign to `array`. Also, the comment about returning an Awkward Array is unnecessary (it's said above and demonstrated below).